### DISCUSSION TITLE
automatic retrieval of tax rates

### DISCUSSION THREAD

**Discussion by noseshimself** [dragotin/kraft#196] [Ideas]
To adapt Kraft to local taxed it might be a good idea to retrieve VAT rates from https://www.abstractapi.com/api/vat-validation-rates-api -- 100 requests per month are free. The service can be used for multiple purposes including verification of EU tax IDs. https://apilayer.com/marketplace/tax_data-api might be interesting, too.

While we are at it: https://vatlayer.com/documentation -- translate EU VAT numbers to entities.

**Comment by dragotin** [comment]
Not really a priority, because the taxes are not changing that fast ;-)
